### PR TITLE
Add install docs for local development

### PR DESCRIPTION
**Pull request by alexgibson**
@pmac r?

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -0,0 +1,200 @@
+GEM
```
Comment (pmac) [inline Gemfile.lock]: This seems like kindof a whole lot to install. All I needed was `jekyll` and `jekyll-paginate` to get it working. I see several themes we don't need as well. Will this help or just be a ton to download?

Comment (alexgibson) [inline Gemfile.lock]: Hmm, I didn't pay it much attention really. All the Gemfile is specifying is the `github-pages` gem. Let me see if I can get it working with the bare bones suggestion

Comment (alexgibson) [inline Gemfile.lock]: Seems to work ok 👍

**Comment by alexgibson** [comment]
@pmac updated - thanks for the suggestion

**Comment by pmac** [comment]
Sweet! Thanks! 💯

**Pull request merged**